iroh/src/client/tags.rs

```diff
@@ -1,5 +1,14 @@
 //! API for tag management.
-
+//!
+//! The purpose of tags is to mark information as important. Currently this is
+//! used for blobs.
```

Comment:
Is it described anywhere *how* this is used for blobs?  Why not give a short description here as an example that gives a general idea of how tags are used?